app/src/main/java/com/example/dine_and_donate/EditProfileActivity.java

```diff
@@ -22,11 +27,11 @@
     private ImageButton mClearNumber;
     private ImageView mProfPic;
     private Button mSaveBtn;
+    private TextView mNumber;
```

Comment:
i'd call this mNumberTextView so it's clearer when using it that you're referring to an actual view and not just a number.